docs/change-log/index.md

```diff
@@ -13,16 +14,181 @@
 - [Limitations](#limitations)
 - [Deprecations](#deprecations)
 
+## v1.3.0
+
+### Client/server matrix
+
+
+
+- The base images mlrun/mlrun:1.3.0 etc. are based on python 3.9. <br>
+      For Iguazio <=3.5.2:
+	1. Configure the Jupyter service with the env variable`JUPYTER_PREFER_ENV_PATH=false`.
+    2. Within the Jupyter service, open a terminal and run:
+     
+	```
+	 conda create -n python39 python=3.9 ipykernel -y
+      conda activate python39
+      ./align_mlrun.sh
+	```
+	  
+- v1.3.0 maintains support for mlrun base images that are based on python 3.7. To differentiate between the images, the images based on python 3.7 have the suffix: `-py37`.
```

Comment:
Should add a comment that the correct version will be automatically chosen for the built-in MLRun images according to the Python version of the mlrun client (so, a 3.7 Jupyter will get the `-py37` images).